cypress/integration/articleSpec.js

```diff
@@ -18,6 +18,12 @@ describe('News Article', () => {
 
     const anchorElement = getElement('header a');
     shouldContainText(anchorElement, 'BBC News');
+
+    it('should have accessibility attributes', () => {
```

Comment:
I think there would be value in wrapping this in a describe block that is something like logo, so it is clear that we are testing a11y of the logo, not just the header.